pkgs/by-name/ko/komac/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, darwin
+, testers
+, komac
+}:
+
+let
+  version = "2.1.0";
+  src = fetchFromGitHub {
+    owner = "russellbanks";
+    repo = "Komac";
+    rev = "v${version}";
+    hash = "sha256-L8UYpNqjRyqf4hPQwD9LaXWu6jYaP34yTwTxcqg+e2U=";
+  };
+in
+rustPlatform.buildRustPackage {
+  inherit version src;
+
+  pname = "komac";
+
+  cargoHash = "sha256-J4QZzbyDr4SDt6LlAy9ZdpqgIufZCZHmOC9eu70wMsM=";
+
+  buildInputs = lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk_11_0.frameworks.SystemConfiguration
```

```suggestion
    darwin.apple_sdk.frameworks.SystemConfiguration
```
Only compatible with 11.0?

---

`apple_sdk` worked on my Intel MacBook 👍